metacat-connector-hive/build.gradle

```diff
@@ -13,11 +13,14 @@
 
 dependencies {
     compile project(':metacat-common-server')
-    compile "org.apache.hive:hive-metastore:${hive_version}"
+    compile("org.apache.hive:hive-metastore:${hive_version}") {
+        exclude group: 'com.google.guava'
+    }
+    compile("com.facebook.presto.hive:hive-apache:${facebook_hive_apache}")
```

Comment:
that's jar bringing in the problem of guava version issue. My guess is that there's a jar having guava inside and the conflicts resolution isn't able to resolve it. We can take a further look if you want.